src/common/runtime/helper/options.ts

```diff
@@ -63,9 +88,9 @@ export type OptionsInfos<Type> = Record<keyof Type, OptionInfo>;
 export const kCTSOptionsInfo: OptionsInfos<CTSOptions> = {
   worker: {
     description: 'run in a worker',
-    parser: optionString,
+    parser: optionWorkerMode,
     selectValueDescriptions: [
-      { value: '', description: 'no worker' },
+      { value: '0', description: 'no worker' },
```

Comment:
Shall we update [docs/intro/developing.md](https://github.com/gpuweb/cts/blob/9db2fcf3ba89e90ca1a13bed512ba54954c8d96e/docs/intro/developing.md?plain=1#L59) as well?